File: app/src/main/java/com/zillennium/utswap/screens/setting/kycScreen/KYCActivity.java
package com.zillennium.utswap.screens.setting.kycScreen;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.zillennium.utswap.bases.BasePassedActivity;
import com.zillennium.utswap.databinding.ActivitySettingKycBinding;

public class KYCActivity extends AppCompatActivity {

    ActivitySettingKycBinding binding;

    @Nullable
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);

        binding = ActivitySettingKycBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());




        // Set Passed Back
        ImageView ivBack = binding.ivBack;
        BasePassedActivity.onPassedBack(KYCActivity.this, ivBack);
    }

}

File: app/src/main/java/com/zillennium/utswap/screens/setting/kyc/KYCActivity.kt
package com.zillennium.utswap.screens.setting.kyc

import android.util.Log
import android.view.View
import com.zillennium.utswap.R
import com.zillennium.utswap.bases.mvp.BaseMvpActivity
import com.zillennium.utswap.databinding.ActivitySettingKycBinding


class KYCActivity :
    BaseMvpActivity<KYCView.View, KYCView.Presenter, ActivitySettingKycBinding>(),
    KYCView.View {

    override var mPresenter: KYCView.Presenter = KYCPresenter()
    override val layoutResource: Int = R.layout.activity_setting_kyc

    override fun initView() {
        super.initView()
        try {
            binding.apply {

//                val view: View = findViewById(R.id.toolbar)

                // Set Passed Back
                ivBack.setOnClickListener {
                    onBackPressed()
                }
            }
            // Code
        } catch (unused: Exception) {
            Log.d("123456789", "hello world")
            // Must be safe
        }
    }
}